flaml/nlp/huggingface/training_args.py

```diff
@@ -102,6 +103,11 @@ def load_args_from_console():
 @dataclass
 class Seq2SeqTrainingArgumentsForAuto(TrainingArgumentsForAuto):
 
+    model_path: str = field(
+        default="t5-small",
+        metadata={"help": "model path for HPO"},
```

Comment:
model path for huggingface transformers?
I'm hoping to see a more informative msg here.